app/controllers/referrals/contact_details_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Referrals
+  class ContactDetailsController < ReferralsController
```

Comment:
Perhaps this should be:

```suggestion
  class ContactDetails::EmailController < ReferralsController
```

with the actions named `edit`/`update`?

Reply:
I was wondering about this approach on the personal details forms. Are we better off with multiple slim controllers and fewer actions? I'm happy either way but will make similar changes [here](https://github.com/DFE-Digital/refer-serious-misconduct/pull/94) for consistency.

Reply:
I think multiple slim controllers with idiomatic `edit/update` actions, plus each one with its own `FormModel` makes sense :+1: